### PR TITLE
added func to return New LogContext With Key value

### PR DESCRIPTION
**Pull request by trivishnu**
Added func to return Context having copy of existing logfields with new key value pair

### CODE REVIEW

File: loggers/fields.go [hunk 1]
```diff
@@ -49,3 +49,15 @@ func FromContext(ctx context.Context) LogFields {
 	}
 	return nil
 }
+
+func GetNewLogContextWithValue(ctx context.Context, key string, value interface{}) context.Context {
+	logfields := FromContext(ctx)
+	newLogFields := make(LogFields)
+	for key, val := range logfields {
```
Comment (NishanthSpShetty) [inline loggers/fields.go]: the key name is reused  here, can we add a test to these functions to verify its not modifying the incoming key?

Comment (trivishnu) [reply loggers/fields.go]: done

Comment (NishanthSpShetty) [reply loggers/fields.go]: test would be good add though

**Review by NishanthSpShetty — APPROVED** [review]
LGTM

**Pull request merged**